specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/notebook.json

```diff
@@ -0,0 +1,461 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved a specified notebook workspace resource.",
+            "schema": {
+              "$ref": "#/definitions/NotebookWorkspace"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2020-03-01/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CosmosDBNotebookWorkspaceGet": {
+            "$ref": "./examples/CosmosDBNotebookWorkspaceGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "NotebookWorkspacesResource"
+        ],
+        "description": "Creates the notebook workspace for a Cosmos DB account.",
+        "operationId": "NotebookWorkspaces_CreateOrUpdate",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookWorkspaceNameParameter"
+          },
+          {
+            "name": "notebookCreateUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NotebookWorkspaceCreateUpdateParameters"
+            },
+            "description": "The notebook workspace to create for the current database account."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Creation of notebook workspace will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/NotebookWorkspace"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2020-03-01/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CosmosDBNotebookWorkspaceCreate": {
+            "$ref": "./examples/CosmosDBNotebookWorkspaceCreate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "NotebookWorkspacesResource"
+        ],
+        "description": "Deletes the notebook workspace for a Cosmos DB account.",
+        "operationId": "NotebookWorkspaces_Delete",
+        "x-ms-long-running-operation": true,
```

Comment:
>"x-ms-long-running-operation": true, [](start = 8, length = 36)

delete is a long running operation but PUT not. Is it intentional?

Reply:
if you see in line 132 here PUT, represented by CreateOrUpdate, is a long-running operation.